client/package.json

```diff
@@ -14,7 +14,7 @@
     "@bpmn-io/properties-panel": "^0.20.1",
     "@bpmn-io/replace-ids": "^0.2.0",
     "@camunda/execution-platform": "^0.3.2",
-    "@camunda/linting": "^0.4.1",
+    "@camunda/linting": "^0.5.0-alpha.3",
```

Comment:
Any reason why it's still in alpha? I mean it sounds unusual to include an alpha dep in a stable release.

Reply:
No particular reason. We could release a 0.5.0. I consider it stable (enough).

Reply:
Let's do that then.